test/migrate/config/migrate-config.test.js

```diff
@@ -11,6 +11,8 @@ const outputPath = path.join(__dirname, outputDir);
 const outputFile = `${outputDir}/updated-webpack.config.js`;
 const outputFilePath = path.join(__dirname, outputFile);
 
+jest.setTimeout(200000);
```

Comment:
Let's use `20000` otherwise CI really can freeze and we should wait a lot of time

Reply:
I copied and pasted from another test not realizing that they are all 200,000 by mistake.